iOS/SampleApp/AppDelegate.m

```diff
@@ -45,4 +47,12 @@ - (void)applicationWillTerminate:(UIApplication *)application {
   // Saves changes in the application's managed object context before the application terminates.
 }
 
+- (OTAcceleratorSession *)acceleratorSession {
+    if (!_acceleratorSession) {
+        _acceleratorSession = [[OTAcceleratorSession alloc] initWithOpenTokApiKey:kApiKey
```

Comment:
You can put it back to `didFinishLaunchingWithOptions`. it's not necessary to do this way.